thewalrus/quantum/photon_number_distributions.py

```diff
@@ -89,3 +90,90 @@ def _convolve_squeezed_state_distribution(s, cutoff=50, padding_factor=2):
     for s_val in s:
         ps = np.convolve(ps, _squeezed_state_distribution(s_val, cutoff_sc))[0:cutoff_sc]
     return ps
+
+
+def total_photon_number_distribution(n, k, s, eta, pref=1.0):
+    r"""Probability of observing a total of :math:`n` photons when :math:`k` identical
+    single-mode squeezed vacua with squeezing parameter s undergo loss by transmission :math:`\eta`.
```

Comment:
I've actually never seen the word "vacua" before. 😄 
```suggestion
    single-mode squeezed vacua with squeezing parameter ``s`` undergo loss by transmission :math:`\eta`.
```

Reply:
Or actually change this to use :math:`s` as below. Same below for my ``n`` to :math:`n` suggestion.

Reply:
https://en.wiktionary.org/wiki/vacuum